### ISSUE TITLE
[Suggestion] Formatting tagEveryone

### ISSUE THREAD

**Issue by t0g3pii** [open-wa/wa-automate-nodejs#2618]
Suggestion for tagEveryone without hideTags
Caption above the Mentioned Users, Tagged Users append a \n and maybe add a optional variable for an Emoji or Symbol before the Mentioned User 

so it looks like
${symVar} @MentionedUser \n

![Screenshot_20220403-203909_WhatsApp.jpg](https://user-images.githubusercontent.com/49153387/161443189-9b428fe1-abb2-4344-a4f2-8d0856f3466f.jpg)

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.33.2](https://github.com/open-wa/wa-automate-nodejs/compare/4.33.1...4.33.2) (2022-04-03)

- 📝 update docs [`6c72f78`](https://github.com/open-wa/wa-automate-nodejs/commit/6c72f78f0fab2ee6b929c8641b50a5119f176c05)
- Release 4.33.2 [`69ec417`](https://github.com/open-wa/wa-automate-nodejs/commit/69ec417f20ef75caff5265b23fa9f06965403270)
- 🔧 update package lock to fix build [`0e47357`](https://github.com/open-wa/wa-automate-nodejs/commit/0e4735733a535107aa97ad2f058be37d303cbf4a)
- ✨ feat: added formatting options for `tagEveryone` #2618 [`729be1c`](https://github.com/open-wa/wa-automate-nodejs/commit/729be1c2d5d91d31ff207c0634d4c312c7f3443f)
- 🏷️ types: `Message.mentionedJidList` to `Contactid[]` [`0b4be9b`](https://github.com/open-wa/wa-automate-nodejs/commit/0b4be9bcdf8c0f5bc5ff845ff602c84f3be9376e)
- 🔧 update releaseit config [`82fbd23`](https://github.com/open-wa/wa-automate-nodejs/commit/82fbd232aa5f9be8c79026c0c14f4cd011fced14)
- 🥚 patches update [`c3970e4`](https://github.com/open-wa/wa-automate-nodejs/commit/c3970e45345b39f7790340298cc118116b328e04)
- 🥚 patches update [`da2755d`](https://github.com/open-wa/wa-automate-nodejs/commit/da2755d4f16c341900c74a7bb342b1d67eafec2b)
- 🥚 patches update [`c9a429c`](https://github.com/open-wa/wa-automate-nodejs/commit/c9a429ca8bb930b7dbc583f5fce62aa8fa8ddee9)
- updated types-only package [`ec9b681`](https://github.com/open-wa/wa-automate-nodejs/commit/ec9b6814a6303ba83ece6853352c18fdb1d0988a)

**Comment by smashah** [comment]
https://docs.openwa.dev/classes/api_Client.Client.html#tagEveryone

try:

```js
await client.tagEveryone(messageId, "test", false, "✨ @mention \n", true)
```
